source/system/Numbers.ooc

```diff
@@ -137,6 +137,7 @@ SizeT: cover from size_t extends ULLong
 SSizeT: cover from ssize_t extends LLong {
 	toString: func -> String { "%u" formatSSizeT(this) }
 }
+PtrDiff: cover from ptrdiff_t extends SSizeT
```

Comment:
Do not use `sizeof(...)`, you will get the size of the pointer to the class. Use `C size` and `Small size`.